src/OrchardCore.Modules/OrchardCore.Markdown/Views/MarkdownBodyPart-Wysiwyg.Edit.cshtml

```diff
@@ -44,7 +44,7 @@
                 $(document).trigger('contentpreview:render');
             });
 
-            if (isRtl)
+            if (isRtl === "true")
```

Comment:
So it's in line with the other strings here (and for some reason single quotes are better in JS, can't remember):

```suggestion
            if (isRtl === 'true')
```